src/api/history/history.controller.js

```diff
@@ -0,0 +1,205 @@
+      },
+      {
+        _id: 2,
+        type: "mileston",
+        data: {
+          name: "Reached 3000 subscribers!",
+          date: "19/08/2018",
+          description: "CJ reached 3000 subscribers!",
+          url: null
+        }
+      },
+      {
+        _id: 2,
+        type: "mileston",
+        data: {
+          name: "Reached 4000 subscribers!",
+          date: "12/09/2018",
+          description: "CJ reached 4000 subscribers!",
+          url: null
+        }
+      },
+      {
+        _id: 2,
+        type: "mileston",
+        data: {
+          name: "Reached 5000 subscribers!",
+          date: "26/09/2018",
+          description: "CJ reached 5000 subscribers!",
+          url: null
+        }
+      },
+      {
+        _id: 2,
+        type: "mileston",
+        data: {
+          name: "Reached 6000 subscribers!",
+          date: "04/10/2018",
+          description: "CJ reached 6000 subscribers!",
+          url: null
+        }
+      },
+      {
+        _id: 2,
+        type: "mileston",
+        data: {
+          name: "Reached 7000 subscribers!",
+          date: "20/10/2018",
+          description: "CJ reached 7000 subscribers!",
+          url: null
+        }
+      },
+      {
+        _id: 4,
+        type: "mileston",
+        data: {
+          name: "Reached 200000 views!",
+          date: "21/10/2018",
+          description: "CJ reached 200000 views!",
+          url: null
+        }
+      },
+      {
+        _id: 2,
+        type: "mileston",
+        data: {
+          name: "Reached 8000 subscribers!",
+          date: "20/11/2018",
+          description: "CJ reached 8000 subscribers!",
+          url: null
+        }
+      },
+      {
+        _id: 2,
+        type: "mileston",
+        data: {
+          name: "Reached 9000 subscribers!",
+          date: "05/01/2019",
+          description: "CJ reached 9000 subscribers!",
+          url: null
+        }
+      },
+      {
+        _id: 4,
+        type: "mileston",
+        data: {
+          name: "Reached 300000 views!",
+          date: "21/01/2019",
+          description: "CJ reached 300000 views!",
+          url: null
+        }
+      },
+      {
+        _id: 2,
+        type: "mileston",
+        data: {
+          name: "Reached 10000 subscribers!",
+          date: "19/02/2019",
+          description: "CJ reached 10000 subscribers!",
+          url: null
+        }
+      },
+      {
+        _id: 4,
+        type: "mileston",
+        data: {
+          name: "Reached 400000 views!",
+          date: "30/03/2019",
+          description: "CJ reached 400000 views!",
+          url: null
+        }
+      },
+      {
+        _id: 2,
+        type: "mileston",
+        data: {
+          name: "Reached 11000 subscribers!",
+          date: "02/04/2019",
+          description: "CJ reached 11000 subscribers!",
+          url: null
+        }
+      }
+    ]
+  }
+};
+
+module.exports = function getHistory(req, res) {
```

Comment:
1. For scalability, define the function independently and export an object with your functions as there will be more than just the `getHistory` function.
2. Follow the project error handling standards so that we have uniform errors throughout. Add `next`, use a try/catch and return `next(error)` in the catch block.